cg/meta/meta.py

```diff
@@ -0,0 +1,28 @@
+from cg.apps.crunchy import CrunchyAPI
+from cg.apps.hermes.hermes_api import HermesApi
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.apps.lims import LimsAPI
+from cg.apps.scout.scoutapi import ScoutAPI
+from cg.apps.tb import TrailblazerAPI
+from cg.apps.vogue import VogueAPI
+from cg.meta.compress import CompressAPI
+from cg.meta.workflow.prepare_fastq import PrepareFastqAPI
+from cg.store import Store
+
+
+class MetaAPI:
```

Comment:
Docstring would be nice, it would be good to know for example if this includes all the available apis

Reply:
This is a really good idea!

Reply:
Thanks, cant say it was mine :yum:

Reply:
done